common/scheme_plugin.go

```diff
@@ -74,13 +89,54 @@ func (s *SchemeServer) GetFormat(args interface{}, resp *AttestationFormat) erro
 	return nil
 }
 
+type SynthKeysArgs struct {
```

Comment:
@thomas-fossati I have opened a generic Issue #136 to address the potential concern I have regarding the way these arguments are handled in scheme_plugin.go